### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by juliettech13**
## Description

Updates the README docs for this repo.

## Type of change

- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] I have selected the correct base branch.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] Any dependent changes have been merged and published in downstream modules.
- [X] I ran all tests with success and extended them if necessary.
- [X] I have updated the ``CHANGELOG.md`` file in the root folder.
- [N/A ] I have tested my code on the test network.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,21 +1,146 @@
-# Aragon OSx
+![Aragon](https://res.cloudinary.com/dacofvu8m/image/upload/v1677353961/Aragon%20CodeArena/osx_blue_logo_lqrvkr.png)
 
-This workspace contains the foundation of the Aragon OSx ecosystem - the human centered approach to DAO's.
+# Aragon OSx Protocsol
```
Comment (Michael-A-Heuer) [inline Readme.md:3]:
```suggestion
# Aragon OSx Protocol
```

File: Readme.md [hunk 2]
```diff
@@ -1,21 +1,146 @@
...
 
 Since the repo is set up as yarn workspace, all the linking is done automatically.
 
+## How the Aragon OSx protocol works
+
+To review the contracts powering the Aragon OSx protocol, feel free to head to `packages/contracts`.
+
+The Aragon OSx protocol architecture is composed of two key sections:
+
+- __Core contracts__: the primitives the end user will interact with. It is composed of mostly 3 sections:
```
Comment (Michael-A-Heuer) [inline Readme.md:32]:
```suggestion
- __Core contracts__: the primitives the end user will interact with. It is composed of mostly 3 parts:
```

File: packages/contracts/README.md [hunk 3]
```diff
@@ -19,7 +25,15 @@ npx solhint 'contracts/**/*.sol'
...
+
+You can find all documentation regarding how to use this protocol in [Aragon's Developer Portal here](https://devs.aragon.org).
+
+## Contributing
+
+If you like what we're doing and would love to support, please review our `CONTRIBUTING_GUIDE.md` [here](https://github.com/aragon/osx/blob/develop/CONTRIBUTION_GUIDE.md). We'd love to build with you.
+
+## Etherscan verification
 
 To try out Etherscan verification, you first need to deploy a contract to an Ethereum network that's supported by Etherscan, such as Ropsten.
```
Comment (Michael-A-Heuer) [inline packages/contracts/README.md:38]:
```suggestion
To try out Etherscan verification, you first need to deploy a contract to an Ethereum testnet that's supported by Etherscan, such as [goerli](https://goerli.etherscan.io) or [sepolia](https://sepolia.etherscan.io).
```

File: Readme.md [hunk 4]
```diff
@@ -1,21 +1,146 @@
...
+    - **DAO contract:** the main contract of our core. It holds a DAO's assets and possible actions.
+    - **Permissions**: govern interactions between the plugins, DAOs, and any other address - allowing them (or not) to execute actions on behalf of and within the DAO.
+    - **Plugins**: base templates of plugins to build upon.
+- __Framework contracts__: in charge of creating and registering each deployed DAO or plugin. It contains:
+    - **DAO and Plugin Factories**: creates DAOs or plugins.
```
Comment (mathewmeconry) [inline Readme.md:37]:
```suggestion
    - **DAO and Plugin Repository Factories**: creates DAOs or plugins.
```
With our `PluginRepoFactory` we create plugin repositories that are registered in the `PluginRepoRegistry`. Plugin repositories are the contracts holding the information about the plugin and its versions

File: Readme.md [hunk 5]
```diff
@@ -1,21 +1,146 @@
...
+    - **DAO and Plugin Registries**: registers into our protocol those DAOs or plugins.
+    - **Plugin Processor:** installs and uninstalls plugins into DAOs.
```
Comment (mathewmeconry) [inline Readme.md:39]:
```suggestion
    - **Plugin Setup Processor:** installs and uninstalls plugins into DAOs.
```

It processes the plugin setups that preprare the plugin for installation/update/deletion

File: Readme.md [hunk 6]
```diff
@@ -1,21 +1,146 @@
...
+
+Additionally to those two sections, we have developed several plugins DAOs can easily install upon creation. These are:
+
+- __Token Voting plugin__: enables token holders to vote yes, no or abstain on incoming DAO proposals
+- __Multisig plugin__: enables DAO governance based on approval from a pre-defined members list.
+- __Addresslist Voting plugin__: enables a pre-defined set of addresses to vote yes, no or abstain in a "one person, one vote" mode
```
Comment (mathewmeconry) [inline Readme.md:45]:
```suggestion
- __Addresslist Voting plugin__: enables a pre-defined set of addresses to vote yes, no or abstain in a "one address, one vote" mode
```
A person can have multiple addresses

File: Readme.md [hunk 7]
```diff
@@ -1,21 +1,146 @@
...
+- __Admin plugin__: enables full access to an account needing to perform initial maintenance tasks without unnecessary overhead
+
+Let's dive into more detail on each of these sections.
+
+### Core Contracts
+
+The *Core Contracts* describe how every DAO generated in the Aragon OSx protocol will be set up. It is extremely lean and by design and constitutes the most critical aspects of our architecture.
```
Comment (mathewmeconry) [inline Readme.md:52]:
```suggestion
The *Core Contracts* describe how every DAO generated by the Aragon OSx protocol will be set up. It is extremely lean and by design and constitutes the most critical aspects of our architecture.
```

File: Readme.md [hunk 8]
```diff
@@ -1,21 +1,146 @@
...
+1. **The DAO contract:** The DAO contract is where the **core functionality** of the DAO lies. It is the contract in charge of:
+    - Representing the identity and metadata of the DAO (ENS name, logo, description, other metadata)
+    - Holding and managing the treasury assets
+    - Executing arbitrary actions to:
+        - transfer assets
+        - call its own functions
+        - call functions in external contracts
+    - Providing general technical utilities like callback handling and others
+2. **Permissions:** Permissions are an integral part of any DAO and the center of our protocol architecture. The Permissions manager **manages permissions for the DAO** by specifying which addresses have permission to call distinct functions on contracts associated with your DAO. This Permissions manager lives inside the DAO contract.
+3. **Plugins**: Any custom functionality can be aded or removed through plugins, allowing you to **fully customize your DAO**. You'll find some base templates of plugins within the `plugins` folder of the *Core Contracts*. Some examples of plugins that DAOs could install are:
```
Comment (mathewmeconry) [inline Readme.md:65]:
```suggestion
3. **Plugins**: Any custom functionality can be added or removed through plugins, allowing you to **fully customize your DAO**. You'll find some base templates of plugins within the `plugins` folder of the *Core Contracts*. Some examples of plugins that DAOs could install are:
```

File: Readme.md [hunk 9]
```diff
@@ -1,21 +1,146 @@
...
+    - Governance (e.g., token voting, one-person one-vote)
```
Comment (mathewmeconry) [inline Readme.md:66]:
```suggestion
    - Governance (e.g., token voting, one-address one-vote)
```

File: Readme.md [hunk 10]
```diff
@@ -1,21 +1,146 @@
...
+    - Asset management (e.g., ERC-20 or NFT minting, token streaming, DeFi)
+    - Membership (governing budget allowances, gating access, curating a member list)
```
Comment (mathewmeconry) [inline Readme.md:68]:
```suggestion
    - Membership (governing budget allowances, access gating, curating a member list)
```

File: Readme.md [hunk 11]
```diff
@@ -1,21 +1,146 @@
...
+
+The following graphic shows how an exemplary DAO setup, where the
```
Comment (mathewmeconry) [inline Readme.md:70]: `where the ?`

**Review by mathewmeconry — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 7 above)
Comment (brickpop) [inline Readme.md]:
```suggestion
The *Core Contracts* describe how every DAO generated in the Aragon OSx protocol will be set up. It is extremely lean by design and constitutes the most critical aspects of our architecture.
```

File: Readme.md (hunk 11 above)
Comment (brickpop) [inline Readme.md:70]:
```suggestion
The following graphic shows an exemplary DAO setup:
```

**Review by brickpop — APPROVED** [review]

File: Readme.md [hunk 12]
```diff
@@ -1,21 +1,146 @@
...
+- __Token Voting plugin__: enables token holders to vote yes, no or abstain on incoming DAO proposals
+- __Multisig plugin__: enables DAO governance based on approval from a pre-defined members list.
+- __Addresslist Voting plugin__: enables a pre-defined set of addresses to vote yes, no or abstain in a "one address, one vote" mode
+- __Admin plugin__: enables full access to an account needing to perform initial maintenance tasks without unnecessary overhead
+
+Let's dive into more detail on each of these sections.
+
+### Core Contracts
+
+The *Core Contracts* describe how every DAO generated by the Aragon OSx protocol will be set up. It is extremely lean and by design and constitutes the most critical aspects of our architecture.
```
Comment (brickpop) [inline Readme.md:52]:
```suggestion
The *Core Contracts* describe how every DAO generated by the Aragon OSx protocol will be set up. It is very lean by design and constitutes the most critical aspects of our architecture.
```

**Review by brickpop — APPROVED** [review]

**Pull request merged**